temporian/implementation/numpy/data/event.py

```diff
@@ -205,7 +209,9 @@ def from_dataframe(
                 for feature in feature_columns
             ]
 
-        numpy_sampling = NumpySampling(index=index_names, data=sampling)
+        numpy_sampling = NumpySampling(
+            index=index_names, data=sampling, is_unix_timestamp=True
```

Comment:
Can the timestamps be provided as a float or int column? If so, I think the is_unix_timestamp should be False.

Reply:
Good catch, will fix and add test